go/engine/identify2_with_uid.go

```diff
@@ -343,23 +343,29 @@ func (e *Identify2WithUID) runIdentifyUI(ctx *Context) (err error) {
 	// fingerprints and the user's UID and username.
 	e.remotesReceived = e.them.BaseProofSet()
 
+	e.G().Log.Debug("| IdentifyUI.Start(%s)", e.them.GetName())
```

Comment:
Would there be any value in also adding a string that would identify each `runIdentifyUI` call?  In case there are multiple overlapping calls for the same username.  Something like:

    | IdentifyUI.Start(max) [abcd]
    | IdenitfyUI.ReportLastTrack(max) [xyz0]
    | IdentifyUI.ReportLastTrack(max) [abcd]

etc.

?